iosxr_iso2vbox.py

```diff
@@ -65,38 +63,295 @@
 import time
 import subprocess
 import getpass
-import shlex
 import argparse
 from argparse import RawDescriptionHelpFormatter
 import re
+import logging
+import pexpect
 
 # Telnet ports used to access IOS XR via socat
-console_port = 65000
-aux_port = 65001
+CONSOLE_PORT = 65000
+AUX_PORT = 65001
 
+logger = logging.getLogger(__name__)
 
-def run(command_string, debug=False):
+
+def set_logging():
+    '''
+    Set basic logging format.
+    '''
+    FORMAT = "[%(filename)s:%(lineno)s - %(funcName)20s() ] %(message)s"
```

Comment:
whitespace between `()` and `]` intentional here?
Can you provide some examples of log output with this format?